config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
+worker_shutdown_timeout 240s;
+
+events {
+  multi_accept        on;
+  worker_connections  16384;
+  use                 epoll;
+}
+
 http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
+  aio       threads;
+  aio_write on;
+
   server_tokens           off;
   sendfile                on;
   tcp_nopush              on;
+  tcp_nodelay             on;
```

Comment:
Another performance improvement which can save 200ms for small requests, see [this article](https://www.netguru.com/codestories/nginx-tutorial-performance).